python/cudf_polars/cudf_polars/dsl/ir.py

```diff
@@ -1554,13 +1554,20 @@ def __init__(self, schema: Schema, name: str, options: Any, df: IR):
             self.options = (
                 tuple(indices),
                 tuple(pivotees),
-                (variable_name, schema[variable_name]),
-                (value_name, schema[value_name]),
+                variable_name,
+                value_name,
             )
-        self._non_child_args = (name, self.options)
+        self._non_child_args = (schema, name, self.options)
+
+    def get_hashable(self) -> Hashable:
```

Comment:
```suggestion
    def get_hashable(self) -> Hashable:  # pragma: no cover; Needed by experimental
```
Pretty sure this is lowering test coverage.

Reply:
I introduced basic testing for all executors independent of `--executor` pytest argument to ensure 100% coverage always.

Reply:
See https://github.com/rapidsai/cudf/pull/17262/commits/9b78d8ff2ea0e465875e8c0b1ccd193ad6e9986b .